telemetry/flow-enricher/cmd/flow-enricher/main.go

```diff
@@ -88,11 +92,21 @@ func main() {
 		os.Exit(1)
 	}
 
+	networkConfig, err := config.NetworkConfigForEnv(os.Getenv("DZ_ENV"))
+	if err != nil {
+		logger.Error("error getting network config", "error", err)
+		os.Exit(1)
+	}
+
+	serviceabilityClient := serviceability.New(rpc.New(networkConfig.LedgerPublicRPCURL), networkConfig.ServiceabilityProgramID)
```

Comment:
I added a wrapper around the RPC client that includes a few retries with backoff if you want to use that here.

Something like (note `rpc` package is a different import, from our `tools/solana/pkg/rpc` dir):
```go
rpcClient := rpc.NewWithRetries(networkConfig.LedgerPublicRPCURL, nil)
serviceabilityClient := serviceability.New(rpcClient, networkConfig.ServiceabilityProgramID)
```